### PR TITLE
6245 additional info

### PR DESCRIPTION
**Pull request by Cooryd**
Closes elifesciences/issues#6245
Closes elifesciences/issues#6306

Will confirm with @chugginselifesciences whether this needs feature flag wrapping or not

### CODE REVIEW

**Comment by chugginselifesciences** [comment]
@Cooryd Yes this PR relates to a ticket for releasing new information about Preprint Review, so is something intended to be "launched" at the same time as press release, editorial etc. currently planned for Dec 1st. Thanks

**Comment by Cooryd** [comment]
TODO: look into top spacing make consistent with login page & articleType page

**Review by ilya-zhuravel — APPROVED** [review]

File: src/dashboard/components/SubmissionEntry.tsx [hunk 1]
```diff
@@ -37,10 +37,7 @@ const SubmissionEntry: React.FC<Props> = ({
                 className={`submission-entry__link submission-entry__link--${status}`}
                 to={submission.lastStepVisited}
                 onClick={(e: React.MouseEvent): void => {
-                    if (
-                        status !== 'continue_submission' ||
-                        (!submission.articleType || submission.articleType === '')
-                    ) {
+                    if (status !== 'continue_submission' || !submission.articleType || submission.articleType === '') {
```
Comment (NuclearRedeye) [inline src/dashboard/components/SubmissionEntry.tsx:40]: Just want to double check, the logic here has changed slightly from 'This or one of these'  to 'this or this or this'. Was that intentional?

**Review by NuclearRedeye — APPROVED** [review]
Changes seem fine, noticed that logic changed in one statement so just want you to double check that was intentional. If so, fine by me to merge.

File: src/dashboard/components/SubmissionEntry.tsx (hunk 1 above)
Comment (Cooryd) [reply src/dashboard/components/SubmissionEntry.tsx:40]: I think the end logic here remains the same. Regardless I've tested manually and it all works as intended so I'm happy for it to go in as is.

**Pull request merged**